collector/src/execute.rs

```diff
@@ -182,6 +185,7 @@ impl Profiler {
             // is rejected because it can't be used with the `profiler`
             // subcommand. (It's used with `bench_local` instead.)
             "perf-stat" => Err(anyhow!("'perf-stat' cannot be used as the profiler")),
+            "xperf-stat" => Err(anyhow!("'xperf-stat' cannot be used as the profiler")),
```

Comment:
Should we explain why these profilers can't be used in the error message?

Reply:
I'm not entirely sure why this isn't allowed. I think technically it should work but the results will just be the statistics captured during the run which is probably not very useful for profiling (vs benchmarking).